src/panels/lovelace/cards/hui-entities-card.ts

```diff
@@ -0,0 +1,191 @@
+import { html, LitElement, PropertyDeclarations } from "@polymer/lit-element";
+
+import "../../../components/ha-card.js";
+import "../components/hui-entities-toggle.js";
+
+import { fireEvent } from "../../../common/dom/fire_event.js";
+import { DOMAINS_HIDE_MORE_INFO } from "../../../common/const.js";
+import { HassLocalizeLitMixin } from "../../../mixins/lit-localize-mixin";
+import { LovelaceCard, LovelaceConfig } from "../types.js";
+import createRowElement from "../common/create-row-element.js";
+import computeDomain from "../../../common/entity/compute_domain.js";
+import processConfigEntities from "../common/process-config-entities";
+import { HomeAssistant } from "../../../types.js";
+import { EntityConfig, EntityRow } from "../entity-rows/types.js";
+
+interface ConfigEntity extends EntityConfig {
+  type?: string;
+  secondary_info: "entity-id" | "last-changed";
+  action_name?: string;
+  service?: string;
+  service_data?: object;
+  url?: string;
+}
+
+interface Config extends LovelaceConfig {
+  show_header_toggle?: boolean;
+  title?: string;
+  entities: ConfigEntity[];
+}
+
+class HuiEntitiesCard extends HassLocalizeLitMixin(LitElement)
+  implements LovelaceCard {
+  protected _hass?: HomeAssistant;
+  protected config?: Config;
+  protected configEntities?: ConfigEntity[];
+
+  set hass(hass) {
+    this._hass = hass;
+    if (this.shadowRoot) {
```

Comment:
shadowRoot is attached in the constructor so no need for this check. Instead use `this.shadowRoot!.querySelectorAll`